TableView/Controllers/DetailController.swift

```diff
@@ -54,11 +54,10 @@ class DetailController: UIViewController {
         super.viewDidLoad()
     
         viewForOpacity.opacityGradient()
-        textLabel.text = modelEntity?.getDescript() ?? "We don`t have description for this ATD"
-        title = modelEntity?.getTitle() ?? "Default ATD"
-        if modelEntity?.getLink() == nil {
+        textLabel.text = modelEntity.descript
+        title = modelEntity.title
+        if modelEntity?.link == nil {
```

Comment:
`modelEntity` declared as `var modelEntity: EntityDataProtocol!` - no need in `?`